src/types/FateBytes.js

```diff
@@ -12,7 +12,13 @@ const toByteArray = (value) => {
         return HexStringToByteArray(value)
     }
 
-    return Int2ByteArray(BigInt(value))
+    try {
```

Comment:
shouldn't this be reverted as well ? I don't feel it was necessary to comment on each error message rewrite

Reply:
The general idea is to show complete error message in case if we wasn't able to convert value to byte array.

Number is the less obvious representation of byte array, can we replace this validation with `Array.isArray` to show error like "is not an array" instead "can't convert to BigIng" in case we remove the complete error message?

We need an agreement on developer friendlies. SDK before was too developer friendly, I'm just trying to make the difference not so significant (on your side).

Reply:
I’m more or less against changing of error messages. Additional validation  should be fine. It’s also fine if SDK strive to be developer friendly, however this lib is more low level and doesn’t target non experienced devs. It should be focused on specification conformance, clean code, maintability (that includes minimal solid API) and performance